rfcs/72-editor.md

```diff
@@ -0,0 +1,508 @@
+# Feature Name: `editor-foundation`
+
+## Summary
+
+This RFC outlines the foundation for Bevy's editor. It describes its basic
+components and how they will interact with Bevy's engine. Following this RFC
+will result in a minimum viable product Bevy Editor.
+
+This RFC does NOT cover the particular user interface (UI) design for the
+editor. Instead, it provides the building blocks for a proper UI design, with a
+minimal usable interface to place these components in context.
+
+## Motivation
+
+Unlike most major game engines, Bevy lacks an interactive editor. This lack has
+three major downsides:
+
+1. it increases the barrier to entry for potential new developers;
+2. this forces scene designers to learn to create scenes with code; and
+3. developers must rebuild their project before seeing any changes they make.
```

Comment:
The ";" make this really hard to read.

Reply:
oops, those used to be multi-part clauses, I can change them to be independent sentences.